addons/web/static/src/views/fields/date/date_field.js

```diff
@@ -10,6 +10,13 @@ import { standardFieldProps } from "../standard_field_props";
 import { Component } from "@odoo/owl";
 
 export class DateField extends Component {
+    setup() {
+        /**
+         * The last value that has been commited to the model.
+         * Not changed in case of invalid field value.
```

Comment:
the comment (the second part at least, doesn't apply here)

Reply:
It is, because in datepicker, we only call the onUpdateInput if the format is correct.

Reply:
indeed :+1: